docs/reference/cooperative_groups_reference.rst

```diff
@@ -0,0 +1,55 @@
+.. meta::
```

Comment:
See the ticket. 
Included a document with changes to apply.

Reply:
Fixed.